src/main/scala/cromwell/backend/provider/local/LocalBackend.scala

```diff
@@ -195,7 +196,7 @@ class LocalBackend(task: TaskDescriptor) extends BackendActor with StrictLogging
     */
   private def buildDockerRunCommand(image: String): String = {
     val dockerVolume = "-v %s:%s"
-    val inputFolderList = extractFolderFromInput(task.inputs.map(_._2).toList)
+    val inputFolderList = extractFolderFromInput(task.inputs.values.toList)
```

Comment:
This change should not happen. Old code was looking for values of the values. 
With the change you did I will look just for values.